vendordeps/REVLib.json

```diff
@@ -4,7 +4,7 @@
     "version": "2023.1.3",
     "uuid": "3f48eb8c-50fe-43a6-9cb7-44c86353c4cb",
     "mavenUrls": [
-        "https://maven.revrobotics.com/"
+      "https://maven.revrobotics.com/"
```

Comment:
Should be 4 space indentation. The file shouldn't have changed though.

Reply:
@Colepng